content/en/cloudprem/install/aws_eks.md

```diff
@@ -141,6 +141,16 @@ echo ""
      # All indexes created in CloudPrem are stored under this location.
      default_index_root_uri: s3://<BUCKET_NAME>/indexes
 
+    # Reverse connection configuration
+    # When enabled, CloudPrem initiates the connection to Datadog, eliminating the need for public ingress.
+    # This is useful for environments with strict network policies or when you prefer not to expose CloudPrem publicly.
+    # cloudprem:
+    #   enable_reverse_connection: true
+    #   site: "datadoghq.com"
+    #   dd_api_key: "${DD_API_KEY}"
```

Comment:
we should maybe add that this env var should be filled by e.g. a secretRef statement